container/runx11docker.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+SCRIPT_DIR=$(dirname $(readlink -f $0))
+SRC_DIR=$SCRIPT_DIR/../randomwallpaper@iflow.space
+DST_DIR=/home/dev/.local/share/gnome-shell/extensions/randomwallpaper@iflow.space
+
+if [ -z "$1" ]
+  then
+    echo "$(basename $0): Provide your docker image as an argument"
+    exit 22
+fi
+
+echo "$(basename $0): You might have to move the X window around before GNOME is fully loaded"
+sleep 3
+
+$SCRIPT_DIR/x11docker/x11docker \
+	--desktop \
+	--init=systemd \
+	--user=RETAIN \
```

Comment:
This should be enough as a first workaround to add the journal to the container output.  
I tested it locally and it works. The output is not limited to gnome-shell and GJS but it seems to be clean enough to work with.

```suggestion
	--user=RETAIN \
	--runasuser="journalctl -f &" \
```

Reply:
Very clever. Works for me :+1: